templates/consulting/sidebar-tabs.html

```diff
@@ -0,0 +1,36 @@
+<script>
+    // This script is used to handle the toggling of the sidebar menu in the consulting template.
+    // It sets up event listeners for the toggle buttons and manages the aria-expanded attributes.
+    
+    document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Does the [tabbed content script](https://github.com/canonical/canonical.com/blob/2c3e6faf43a9710f99df8a7ffc7df04c51e14e08/static/js/tabbed-content.js) not work with these tabs?

Reply:
Yes I didn't know it was there. I have replaced it to use the tabbed content now